frontend/src/components/Banner.tsx

```diff
@@ -54,6 +57,16 @@ export const css = stylesheet({
   refreshButton: {
     backgroundColor: color.background,
   },
+  troubleShootingLink: {
+    alignItems: 'center',
+    color: color.theme,
+    cursor: 'pointer',
+    display: 'flex !important',
+    flexShrink: 0,
+    fontWeight: 'bold',
+    padding: spacing.units(-4),
+    textDecoration: 'underline',
```

Comment:
I wonder how did you come up with these.

One looks redundant, e.g. `cursor: 'pointer'`.
although display flex + alignItems center + flexShrink 0 works. Would just `text-align: center` be enough? https://www.w3schools.com/cssref/pr_text_text-align.ASP

I didn't test these on your css, so there may be problems. Recommend putting your reference/reasoning next to these. Because css easily becomes a myth that others cannot understand.

Reply:
Thank you, that is a very good catch. I cleaned up the css and removed the extra tags, they are not needed as you pointed out.